packages/fxa-payments-server/src/components/NewUserEmailForm/index.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { NewUserEmailForm } from './index';
+
+describe('NewUserEmailForm test with l10n', () => {
+  it('renders as expected', () => {
+    const { queryByTestId } = render(
+      <NewUserEmailForm onBlurInitial={() => {}} onBlurConfirm={() => {}} />
+    );
+    const form = queryByTestId('newuseremailform');
+    expect(form).toBeInTheDocument();
+  });
```

Comment:
May be helpful to add a test that asserts that when the first/second fields are blurred in the form, that `onBlurInitial` (ideally renamed to avoid confusion) and `onBlurConfirm` are called.